### PR TITLE
Fixes spelling error in opened cryogenic pods.

### PR DESCRIPTION
**Pull request by BeeSting12**
:cl: BeeSting12
spellcheck: Occupand ---> Occupant on opened cryogenic pods.
spellcheck: Cyrogenic ---> Cryogenic on opened cryogenic pods.
/:cl:

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]

**Comment by BeeSting12** [comment]
I'm grinding for GBP right now

File: code/game/objects/structures/showcase.dm [hunk 1]
```diff
@@ -47,7 +47,7 @@
 
 /obj/structure/showcase/machinery/oldpod/used
 	name = "opened cyrogenic pod"
-	desc = "Cyrogenic pod that has recently discharged its occupand. The pod appears non-functional."
+	desc = "Cyrogenic pod that has recently discharged its occupant. The pod appears non-functional."
```
Comment (Mark9013100) [inline code/game/objects/structures/showcase.dm]: "Cyrogenic pod that has recently discharged its occupant. The pod appears non-functional." <- The second part of the sentence doesn't really mesh well imo. Also adding an A at the start of the first sentence might be good too.

**Comment by BeeSting12** [comment]
Someone misspelled cryogenic, who do I yell at? @Mark9013100 If you have any ideas on how to fix the sentence then I'll do it, otherwise I'm just spellchecking it.

**Comment by Mark9013100** [comment]
The second part could probably be reworded to ', it appears non-functional now.' or something along those lines.

**Comment by BeeSting12** [comment]
I think it's fine as is.

**Pull request merged**